ifrs17/Import/2ImportScope-PresentValue.ipynb

```diff
@@ -890,13 +897,13 @@
                 "\n                            .MonthlyAmortizationFactors",
                 "\n                            .Skip(shift)",
                 "\n                            .Take(timeStep)",
+                "\n                            .DefaultIfEmpty()",
                 "\n                            .Aggregate(1d, (x, y) => x * y);",
+                "\n                            ",
                 "\n    [NotVisible] string EconomicBasis => GetContext();",
                 "\n",
                 "\n    string EstimateType => EstimateTypes.F;",
-                "\n    double Value => Math.Abs(amortizedFactor - 1d) > Precision",
-                "\n                        ? 1d - amortizedFactor",
-                "\n                        : 1d;",
+                "\n    double Value => 1d - amortizedFactor;",
```

Comment:
The run off case would produce a MonthlyAmortizationFactors of 0 and length 1. Right? 
So that Value = 1.

Reply:
.DefaultIfEmpty() will give a double[]{0d} if the skip().take() gives an empty. So, yes, it will producea MonthlyAmortizationFactors of 0 and length 1, thus Value of 1. All correct.